lua/cfc_disconnect_interface/client/cl_interface.lua

```diff
@@ -126,7 +126,8 @@ local function addTitleBar( frame )
     local closeBtnPadding = ( titleBarHeight - closeBtnSize ) / 2
     local closeBtn = vgui.Create( "DImageButton", titleBar )
     closeBtn:SetSize( closeBtnSize, closeBtnSize )
-    closeBtn:SetPos( frameW - closeBtnSize - closeBtnPadding, closeBtnPadding )
+    -- TODO: is this the correct usgae of frameH?
```

Comment:
Did you test this change? I'm pretty sure it'll move the button to be too far down, setting Y position to closeBtnPadding means there will be a closeBtnPadding gap between top of frame and the close button.